recipes/server_setup.rb

```diff
@@ -92,6 +103,7 @@
       # Create option string
       force = false
       options = ''
+      force = true if node['gluster']['server']['disks'].empty?
```

Comment:
`df /path/to/dir | tail -1` then check the last part of the array
There is probably other solutions too

Example outside the context of chef
```
[3] pry(main)> `df /etc | tail -1`.chomp.split[-1]
=> "/"
[4] pry(main)> 
```